contrib/autopilot/models.py

```diff
@@ -14,9 +14,15 @@
 # Names model types
 MODEL_NONE = "none"
 MODEL_TRANSFORMER = "transformer"
+MODEL_GNMT = "gnmt"
+
+# Names model test settings
+MODEL_TEST_ARGS_TRANSFORMER = "ttransformer"
+MODEL_TEST_ARGS_GNMT = "tgnmt"
```

Comment:
We shouldn't need `MODEL_TEST_ARGS_TRANSFORMER ` or `MODEL_TEST_ARGS_GNMT` if we already have `MODEL_TRANSFORMER` and `MODEL_GNMT`.